index.html

```diff
@@ -87,6 +87,35 @@
           </div>
         </div>
         <h1 class="heading-48">Component Library</h1>
+        <div class="alert alert--info">
+          <p class="alert__message">
+            The component library is a snapshot of the work found in assets-frontend. Currently assets-frontend is being refactored.
```

Comment:
"This component library is a visual representation of the styles used to build HMRC's services."